### DISCUSSION TITLE
Worlds on friction

### DISCUSSION THREAD

**Discussion by emrodk** [gamerevar/proyecto-comunitario-1-gd#61] [Archivados]
Pensando un poco en la mejor manera de coordinar los juegos sin que se genere mucha fricción en el desarrollo, ya que a la complejidad de organizar un equipo se le van agregando capas a medida que un juego dependa mucho del otro. Entonces la idea va por ese camino, tratar de encontrar una simbiosis que favorezca tanto a los juegos como a la coordinación de ambos equipos.

La idea gira en torno a universos paralelos, primero porque me encanta todo lo que es sci-fi y segundo porque que nos permite posicionar dos protagonistas en dos realidades distintas pero a la vez interconectadas por un mismo hecho. Dos mundos paralelos que están colisionando, un mismo hecho que afecta a cada mundo de manera distinta, uno se encuentra en completo caos y el otro en una tensa calma cómo si se tratara del ojo de una tormenta. Cada mundo tiene sus propios problemas, uno al estar al borde del colapso es muy probable que siente mejor explorar el genero acción/aventura (dónde bien podría tomarse el gameplay del resto de las ideas propuestas) y el otro más calmo acercándose lentamente al final que puede dar más lugar a la exploración o misterio.
Otro punto a favor, es que el final es uno solo y al mismo tiempo diferentes ya que juega mucho la perspectiva de cada personaje, el objetivo que tenga y su propia construcción de la realidad. No habría necesidad de tener un tercer final.

# Worlds on friction: Truth Seeker
* Narrativa: Nadie sabe con exactitud cómo empezó, te levantas un día, miras el teléfono, cientos de vuelos comerciales desaparecidos a lo largo del mundo y ese fue solo el principio… edificios completos desaparecieron de la noche a la mañana! EDIFICIOS COMPLETOS!!...  qué está pasando? Nada tiene sentido…
* Objetivo: Encuentra la verdad
* Género: point and click, mystery, puzzle

# Worlds on friction : On the edge of collapse
* Narrativa: Nadie sabe con exactitud cómo empezó, te levantas un día, miras el teléfono, el mundo está en caos! cientos de vuelos comerciales no identificados estrellándose a lo largo del mundo y ese fue solo el principio… edificios completos aparecieron de la nada! EDIFICIOS COMPLETOS!!... qué está pasando? Nada tiene sentido…
* Objetivo: Sobrevive 
* Género: metroidvania,survival crafting

**Comment by jh62** [comment]
La idea que mas me gusto por ahora y 👍 por el sci-fi 🤣 

Me encantaria poder hacer entre todos un buen point and click, mas si es estilo pixel art.
El segundo, estilo metroidvania, puede funcionar muy bien tambien; ademas de que es el genero que más se votó hasta ahora (o que tuvo mas reacciones/comentarios).

Universos paralelos para mi es algo muy llamativo y a su vez mas facil de justificar en ambos juegos. La historia no se si me copó mucho, pero se le puede buscar la vuelta.

**Reply by emrodk** [reply]
Tal cual mundos paralelos es la clave, después la historia puede ser la que sea por eso no explaye tanto, me acordé de una serie sci-fi que pasaba algo parecido (que no sé si decirla cuenta a spoiler jajaj) y servía muy bien para mostrar como se podría plantear el vínculo entre los dos juegos
